### PR TITLE
Feature final updates

### PR DESCRIPTION
**Pull request by buressjan**

### CODE REVIEW

File: src/components/layout/LineUpHeadlinerLeft.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  genre,
+  igLink,
+  spotifyLink,
+  headliner,
+}) => {
+  return (
+    <div
+      className={`w-full md:w-3/5 ${
+        headliner ? '' : 'max-h-[120px]'
+      } min-h-[120px] md:min-h-[120px] green-shadow uppercase overflow-hidden`}
```
Comment (tomesfilip) [inline src/components/layout/LineUpHeadlinerLeft.tsx:27]: `md:min-h-[120px]` is irrelevant when `min-h-[120px]` is defined already

File: src/components/layout/LineUpHeadlinerRight.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const LineUpHeadlinerRight: React.FC = () => {
+  return (
+    <div className='flex md:w-2/5 bg-[#1D6858] h-[500px] green-shadow overflow-hidden'>
+      <div className='relative'>
+        <div className='absolute bottom-6 left-0 text-white px-2 font-bold '>
+          <span className='text-xl xl:text-2xl'>???</span>
+        </div>
+        <div className='absolute bottom-0 left-0 text-white px-2 text-lg'>
+          ???
```
Comment (tomesfilip) [inline src/components/layout/LineUpHeadlinerRight.tsx:9]: is this intended?

File: src/components/layout/LineUpRight.tsx [hunk 3]
```diff
@@ -5,24 +5,32 @@ import spotify from '../../assets/icons/spotify.png';
...
   igLink,
   spotifyLink,
+  headliner,
 }) => {
   return (
-    <div className='w-full md:w-2/5 bg-[#1D6858] min-h-[120px] md:min-h-[120px] green-shadow uppercase'>
+    <div
+      className={`w-full md:w-2/5 ${
+        headliner ? 'min-h-[200px]' : 'min-h-[120px]'
+      } min-h-[120px] md:min-h-[120px] green-shadow uppercase `}
```
Comment (tomesfilip) [inline src/components/layout/LineUpRight.tsx:27]: md:min-h-[120px] is irrelevant when min-h-[120px] is defined already

File: src/components/shared/Header.tsx [hunk 4]
```diff
@@ -11,14 +11,14 @@ const headerItems = [
...
-  //   text: 'Partneři',
-  // },
   {
     tag: '#kontakt',
     text: 'Kontakt',
   },
+  {
+    tag: '#partneri',
+    text: 'Partneři',
+  },
```
Comment (tomesfilip) [inline src/components/shared/Header.tsx:21]: wouldn't be better to have link `partneri` before `kontakt`? 
on the website, the section `partner` is also before `kontakt` section (that's why)

**Review by tomesfilip — CHANGES_REQUESTED** [review]
really nice, just some minor things to solve.

also, the [f1_fitness.jpg] is present but only the .png version is used, maybe remove this .jpg one so there're no duplicates

File: src/components/layout/LineUpHeadlinerRight.tsx (hunk 2 above)
Comment (buressjan) [reply src/components/layout/LineUpHeadlinerRight.tsx:9]: Yes, but it was just tmp file (until I had the last graphic assets for both Headliners)

**Review by tomesfilip — APPROVED** [review]
GJ

**Pull request merged**